main/src/ca/uwaterloo/flix/api/lsp/provider/completion/KeywordCompleter.scala

```diff
@@ -35,12 +35,15 @@ object KeywordCompleter {
       Completion.KeywordCompletion("eff"      , Priority.Low),
       Completion.KeywordCompletion("enum"     , Priority.High),
       // I
-      Completion.KeywordCompletion("import"   , Priority.Low),
-      Completion.KeywordCompletion("instance" , Priority.High),
+      Completion.KeywordCompletion("import"   , Priority.High),
+      Completion.KeywordCompletion("inline"   , Priority.Low),
```

Comment:
@jaschdoc Will you be making use of this keyword or should we drop it?

Reply:
Drop it for now

Reply:
Dump it. In fact remove the keyword.

Reply:
Do you mean it should also be removed from the lexer and compiler?